src/nxt_conf_validation.c

```diff
@@ -1465,6 +1515,73 @@ nxt_conf_validate(nxt_conf_validation_t *vldt)
     "a number, a string, an array, or an object"
 
 
+
+#if (NXT_HAVE_OTEL)
+inline nxt_int_t
```

Comment:
Remove the `inline`, or simply remove this function altogether is probably better...